### DISCUSSION TITLE
Need some help with this website - menus and photos not working

### DISCUSSION THREAD

**Discussion by snabie23** [gildas-lormeau/SingleFile#1157] [General]
Hello,

I might be not skilled enough to use SingleFile, so please excuse if this is a basic question. But I cannot get SingleFile to save this website correctly. [https://www.nordicnest.com/brands/menu/carrie-lamp-limited-edition/?variantId=515276-01](https://www.nordicnest.com/brands/menu/carrie-lamp-limited-edition/?variantId=515276-01)

When opening the generated save file I cannot for example click through the product photos. When I click them nothing happens. Also there's some menus like 'Facst', 'Reviews' etc. which unfold when use the site online but who do not unfold in the saved website.

I tried a lot of different settings for SingleFile but couldn't get it to work. Is it possible to save the website in a way that the things mentioned still work, and if so, what setting would I need to make in SingleFile?

**Comment by snabie23** [comment]
I tried the suggestion made [here](https://github.com/gildas-lormeau/SingleFile/discussions/916#discussioncomment-2508140) but this doesn't work with the website I want to save. Any help would be appreciated.

**Reply by gildas-lormeau** [reply]
Unfortunately, it means that SingleFile cannot help you.
